tools/depends/target/pythonmodule-pil/Makefile

```diff
@@ -64,9 +66,9 @@ endif
 ifeq ($(OS),android)
 	cd $(PILPATHLIB) && unzip -o ../Pillow-*.egg
 	cd $(PILPATHLIB)/PIL && \
-          sed -i -e 's/import sys/import os, sys /' \
-                 -e '/__file__/ s/_imaging/lib_imaging/g' \
-                 -e 's/pkg_resources.resource_filename(__name__,/os.path.join(os.environ["KODI_ANDROID_LIBS"], /'  _imaging*.py
+          sed -ie 's/import sys/import os, sys /' _imaging*.py && \
```

Comment:
You can't chain sed commands?

Reply:
to do a destructive in place sed on macos, need to do 

```
sed -i  '' '/__file__/ s/_imaging/lib_imaging/g' _imaging.py
```
which doesnt play nice on linux. Working through an alternative without the double backup currently. See what i come up with in a few minutes

Reply:
there you go, i was wrong, all sorted and tested on linux/mac hosts

Reply:
still need the backup postfix (e, but can be anything i guess), as mac requires the explicit -i '' command, where linux doesnt.

Reply:
linux (with the mac required '')
```
sed -i '' -e 's/import sys/import os, sys /' -e '/__file__/ s/_imaging/lib_imaging/g' -e 's/pkg_resources.resource_filename(__name__,/os.path.join(os.environ["KODI_ANDROID_LIBS"], /'  _imaging.py
sed: can't read : No such file or directory

```

macos without the ''
```
sed -i -e 's/import sys/import os, sys /' -e '/__file__/ s/_imaging/lib_imaging/g' -e 's/pkg_resources.resource_filename(__name__,/os.path.join(os.environ["KODI_ANDROID_LIBS"], /'  _imaging.pye 
sed: -e: No such file or directory
```

so we can either backup to a file, or i guess use a variable to set the appropriate -i flag based on build host?

Reply:
I'd go with the variable option